core/include/traccc/clusterization/spacepoint_formation_algorithm.hpp

```diff
@@ -42,16 +42,15 @@ class spacepoint_formation_algorithm
     /// module
     ///
     /// @param measurements_view A collection of measurements
-    /// @param modules_view A collection of modules the measurements link to
-    /// @return A spacepoint container, with one spacepoint for every
-    ///         measurement
+    /// @param dd_view           The detector description
+    /// @return The reconstructed spacepoint collection
     ///
     output_type operator()(
         const measurement_collection_types::const_view& measurements_view,
-        const cell_module_collection_types::const_view& modules_view)
-        const override;
+        const silicon_detector_description::const_view& dd_view) const override;
```

Comment:
We need to be careful in this because our spacepoint formation currently does not work with strip measurement.

Reply:
Sure. See my previous comment.